src/discord_objects.py

```diff
@@ -0,0 +1,115 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Self
+
+
+@dataclass
+class DiscordUser:
+    id: int
+    avatar: str
+    user_name: str
+    global_name: str
+
+    @classmethod
+    def from_dict(cls, user: dict) -> Self:
+        return cls(
```

Comment:
What does cls do? How does this work? I saw that it's something like `self`?

Reply:
Yes, `cls` is the current class, in this case, `DiscordUser`. So, calling `cls(...)` is like calling `DiscordUser(...)`.
https://docs.python.org/3.11/library/functions.html#classmethod

Reply:
Oh, so you're making an instance of DiscordUser? Got it.